doc/nrf/libraries/caf/buttons.rst

```diff
@@ -168,3 +169,8 @@ If :c:struct:`power_down_event` comes while the module is in the ``STATE_ACTIVE`
 Similarly, as in ``STATE_ACTIVE``, in ``STATE_IDLE`` the module enables the GPIO interrupts and waits for the pin state to change.
 However, in ``STATE_IDLE`` the interrupts are enabled only for the subset of buttons that are configured to wake up the application.
 Pressing any of these buttons also invokes :c:struct:`wake_up_event` and sends it to all subscribing modules.
+
+.. note::
+  If system enters off state (indicated by :c:struct:`power_off_event`), a button press triggers an instant system reboot (:c:func:`sys_reboot`).
+  This is done to prevent entering system off state with GPIO interrupts disabled (:ref:`sys_poweroff` call may prevent CAF Buttons module from scanning buttons and reenabling GPIO interrupts).
```

Comment:
```suggestion
  This is done to prevent entering a system-off state with GPIO interrupts disabled (:ref:`sys_poweroff` call may prevent the CAF Buttons module from scanning buttons and reenabling GPIO interrupts).
```
Maybe link to the CAF Buttons module?

Reply:
The note resides in doc page of CAF Buttons module, not sure if we should link in that case (we used not to do it).

Reply:
`a system-off state` -> In other places we used `the system off state`. Can we do it also here?